src/extension/extensionServer.ts

```diff
@@ -235,6 +239,18 @@ export class ExtensionServer implements vscode.Disposable {
                 TargetPlatformHelper.checkTargetPlatformSupport(mobilePlatformOptions.platform);
                 return mobilePlatform.beforeStartPackager()
                     .then(() => {
+                        generator.step("getReactNativeVersion");
+                        return ReactNativeProjectHelper.getReactNativePackageVersionFromNodeModules(
+                            path.resolve(request.arguments.cwd, "node_modules", "react-native")
+                            );
+                    })
+                    .catch(err => {
+                        const noReactNativePackageError = ErrorHelper.getInternalError(InternalErrorCode.ReactNativePackageIsNotInstalled);
+                        generator.addError(noReactNativePackageError);
+                        this.logger.warning(localize("ReactNativePackageIsNotInstalledWarning", "It seems that 'react-native' package is not installed. Please run 'npm install' to install the package."));
+                        throw noReactNativePackageError;
```

Comment:
```suggestion
                        Q.reject(noReactNativePackageError);
```

Reply:
If `Q.reject` or `reject` is used, then the following error message will be displayed:
![image](https://user-images.githubusercontent.com/33267199/66933984-7e5c5980-f042-11e9-995b-81ff55d1280f.png)
If `throw` is used, then the correct error message will be displayed:
![image](https://user-images.githubusercontent.com/33267199/66934166-c1b6c800-f042-11e9-8843-69a7f35e7443.png)